lib/ash/query/aggregate.ex

```diff
@@ -230,6 +239,19 @@ defmodule Ash.Query.Aggregate do
           build_opts -> build_query(target_resource, resource, build_opts)
         end
 
+      # Set bypass context IMMEDIATELY if multitenancy bypass is requested
+      query =
+        if opts.multitenancy == :bypass do
+          query
+          |> Ash.Query.set_tenant(nil)
+          |> Ash.Query.set_context(%{
+            multitenancy: :bypass_all,
```

Comment:
no need to set this key, it will be copied from `shared`

Reply:
Done 🙏🏻 i think i got it from read action